pdr_backend/pdr_dashboard/dash_components/view_elements.py

```diff
@@ -223,7 +224,17 @@ def get_graphs_column_plots_row():
         [
             html.Div(id="accuracy_chart"),
             html.Div(id="profit_chart"),
-            html.Div(id="stake_chart"),
+            html.Div(
+                [
+                    html.Div(id="cost_chart", style={"width": "48%"}),
+                    html.Div(id="stake_chart", style={"width": "48%"}),
+                ],
+                style={
+                    "width": "100%",
```

Comment:
Just a note, we can move the styles to the style.css for HTML tags later. Unfortunately, the className property does not exist in datatables.

Reply:
For that, we would need to define a class or an id